src/libcore/ptr/non_null.rs

```diff
@@ -131,6 +136,7 @@ impl<T: ?Sized> NonNull<T> {
     #[rustc_const_stable(feature = "const_nonnull_cast", since = "1.32.0")]
     #[inline]
     pub const fn cast<U>(self) -> NonNull<U> {
+        // SAFETY: `self` is passed as a pointer and is necessarily initialized
```

Comment:
```suggestion

	        // SAFETY: `self` a `NonNull` pointer which is necessarily non null.

```

Reply:
Good point, my sentence was confusing. I corrected a small typo

Reply:
Fixed in aae8d54